<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SPIMB040OptionsWidget</class>
 <widget class="QWidget" name="SPIMB040OptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&lt;b&gt;Note:&lt;/b&gt; You will have to restart QuickFit, so these settings take full effect!</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>optics setup file directory:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QFEnhancedLineEdit" name="edtOptSetupDir"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>readonly (!!!) optics setup config file directory:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QFEnhancedLineEdit" name="edtOptSetupConfigDirReadonly"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>editable optics setup config file directory:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QFEnhancedLineEdit" name="edtOptSetupConfigDir"/>
   </item>
   <item row="7" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>542</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>St&amp;yle:</string>
     </property>
     <property name="buddy">
      <cstring>cmbStyle</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QComboBox" name="cmbStyle"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>&amp;Stylesheet:</string>
     </property>
     <property name="buddy">
      <cstring>cmbStylesheet</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="cmbStylesheet"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>default acquisition templates directory</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QFEnhancedLineEdit" name="edtOptTemplatesDir"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>default acquisiton scripts directory</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QFEnhancedLineEdit" name="edtOptScriptsDir"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFEnhancedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfenhancedlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
